modules/compute_instance/variables.tf

```diff
@@ -86,3 +86,12 @@ variable "deletion_protection" {
   description = "Enable deletion protection on this instance. Note: you must disable deletion protection before removing the resource, or the instance cannot be deleted and the Terraform run will not complete successfully."
   default     = false
 }
+
+variable "alias_ip_range" {
```

Comment:
```suggestion
variable "alias_ip_ranges" {
```

Reply:
I didn't apply your suggestion via github UI 'cause I have to fix other things.